react/react installer/e257d740-d47d-4794-82f1-3704850bdba9.json

```diff
@@ -0,0 +1,56 @@
+{
+  "implementations": {
+    "es6-node": {
+      "inputFunctions": {
+        "df3107d7-179c-4e60-86b4-323813c9e86e": "const { exec } = require( \"child_process\" );\nconst { projectPath } = data;\nconst { replace } = params;\nlet command = \"npx create-react-app \" + data + \" --use-npm\";\nif (state.running){\n  outputs.bounced(data, tag);\n} else {\n  state.running = true;\n  if ( params.replace === true ) {\n    command = \"rm -rf \" + data + \" && npx create-react-app \" + data + \" --use-npm\";\n  }\n  outputs.stdout( command, tag );\n  exec(command, (error, stdout, stderr) => {\nif (error && (params.replace === false || 'undefined' )) {\n  outputs.error(`Application folder already exists. Set the replace input to true if you wish to reinstall the application from scratch`, tag);\n  outputs.trigger(false, tag);\n} else {\n  outputs.stdout(stdout || \"\", tag);\n  outputs.stderr(stderr || \"\", tag);\n  outputs.error(null, tag);\n  outputs.trigger(true, tag);\n}\n  state.running = false;\n  });\n}"
+      }
+    }
+  },
+  "interface": {
+    "inputs": {
+      "df3107d7-179c-4e60-86b4-323813c9e86e": {
+        "type": "string",
+        "name": "projectPath",
+        "description": "Full file path of react application."
+      },
+      "32afb2d1-73c5-437c-a82a-8ec997b9a9c1": {
+        "type": "boolean",
+        "name": "replace",
+        "description": "Should the node delete and reinstall the application?\n\nboolean: defaults to false."
```

Comment:
I'd add to the description that "Works as static parameter only." 
As it won't work if set by a signal arriving too late.

Reply:
added - Use as a static parameter only.